### ISSUE TITLE
Can't move more than one plate in time

### ISSUE THREAD

**Issue by DKurilo** [rsurkin/game15#1]
I have:
|1 2 3 _|
I want to move them right by clicking on 1.
And it's not working.

**Comment by rsurkin** [comment]
yep, I found this issue, while i play myself, hope will have time at the end of the week to fix it,
